libvast/src/system/spawn_source.cpp

```diff
@@ -63,7 +66,30 @@ expected<actor> spawn_source(local_actor* self, options& opts) {
   auto grd = caf::detail::make_scope_guard([&] { opts.params = r.remainder; });
   // Parse format-specific parameters, if any.
   actor src;
-  if (format == "pcap") {
+  if (format == "broker") {
+#ifndef VAST_HAVE_BROKER
+    return make_error(ec::unspecified, "not compiled with broker support");
+#else
+    auto endpoint = "localhost:9999"s;
+    auto listen = false;
+    auto topic = ""s;
+    r = r.remainder.extract_opts({
+      // TODO: should we reuse the -r instead of adding a new flag -e? The
```

Comment:
Depends on how much of Broker internals we wish to hide. I'm fine with `-e` and `-t`, though. Users of the source should know what Broker endpoints and topics are and I'm in favor of using domain-specific names.